src/js/stores/MetronomeStore.js

```diff
@@ -130,7 +130,7 @@ class MetronomeStore extends EventEmitter {
           this.emit(METRONOME_JOB_DELETE_SUCCESS, action.jobID);
           break;
         case REQUEST_METRONOME_JOB_DETAIL_SUCCESS:
-          this.data.jobDetail[action.jobID] = action.data;
+          this.data.jobs = [].concat(this.data.jobs, action.data);
```

Comment:
This can be also written using spread operator `[...this.data.jobs, ...action.data]`